.github/ISSUE_TEMPLATE/new_chapter.yml

```diff
@@ -0,0 +1,67 @@
+name: "\U0001F4D6 New Chapter Template 2"
+description: Template for proposing a new chapter
+title: "<title>"
+labels: book
+body:
+- type: markdown
+  attributes:
+    value: "Please complete the following sections to propose a new chapter. If you have the necessary permissions, please also assign the appropriate label (or labels) to your issue, otherwise a _Turing Way_ team member will assign labels later."
+- type: input
+  id: title
+  attributes:
+    label: Title
+    description: |
+      Please provide a proposed title
+  validations:
+    required: true
+- type: input
+  id: guide
+  attributes:
+    label: Guide
+    description: |
+      Which Guide will your chapter be a part of? Please reference the [book](https://the-turing-way.netlify.app/welcome.html)'s table of contents to see which Guides make up the book
+  validations:
+    required: true
+- type: input
+  id: branch
+  attributes:
+    label: Branch
+    description: |
+      Please provide the name of the branch you are working from. You can provide this later if you haven't started yet!
```

Comment:
Will most people be working from forks when they start to write?  Should this field account for that in some way?